images/vuejs/Debian/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM debian:latest
```

Comment:
Do we want to have this set to latest, or should we pin it to a current version? It might help us avoid an update breaking the image down the line.

Reply:
@jshoberg yes, it makes sense. What about version 12.4 from https://hub.docker.com/_/debian ?

Reply:
That looks pretty good, if that version runs on your end looks good to me!